jupytext/languages.py

```diff
@@ -3,7 +3,7 @@
 # Languages that may appear as magic instructions in Python notebooks
 _JUPYTER_LANGUAGES = ['R', 'bash', 'sh', 'python', 'python2', 'python3', 'javascript', 'js', 'perl',
                       'html', 'latex', 'markdown', 'pypy', 'ruby', 'script', 'svg', 'writefile',
-                      'matlab', 'octave', 'idl']
+                      'matlab', 'octave', 'idl', 'scala']
```

Comment:
I think we should add `'scala'` here only if there exists a jupyter magic command with the same name. Is that the case? Do you see any trouble with scala notebooks represented as Markdown files when you don't add  `'scala'` here?

Reply:
Hello mwouts, 

thank you for your fast reply.

When we do not define scala in the list of _JUPYTER_LANGUAGES, then *.scala files can not be opened as a notebook in Jupyterhub. You are limited to work only on source files...
![image](https://user-images.githubusercontent.com/18640690/59340682-be16db80-8d06-11e9-8d2f-1bc38044d5ea.png)

Reply:
You're welcome.

Well the classification of `.scala` files as notebooks comes from having added a `.scala` entry in the `_SCRIPT_EXTENSIONS` dictionary, not from `JUPYTER_LANGUAGES`. But anyway I see that there is indeed a `%%spark` magic for Python notebooks provided by [sparkmagic](https://github.com/jupyter-incubator/sparkmagic ), so it is OK to add `scala` there as well.

Do you think you will be able to contribute a sample scala notebook (does not need to be complex)? It should go to `tests/notebooks/ipynb_scala`, and then you will have to modify `tests/test_mirror.py` to add the following:
```
@pytest.mark.parametrize('nb_file', list_notebooks('ipynb_scala'))
def test_ipynb_to_scala(nb_file):
    assert_conversion_same_as_mirror(nb_file, 'scala', 'ipynb_to_script')


@pytest.mark.parametrize('nb_file', list_notebooks('ipynb_scala'))
def test_ipynb_to_scala_percent(nb_file):
    assert_conversion_same_as_mirror(nb_file, 'scala:percent', 'ipynb_to_percent')
```

Running the tests will generate the text representations - I need your help here to make sure these are legitimate scala files! Please let me know if you need more details. And thanks again for your contribution!

Reply:
Added a simple scala notebook to see if the corresponding tests work...

We are using the Apache Toree plugin (https://toree.incubator.apache.org/), which have a lot of scala magics:
https://github.com/apache/incubator-toree/blob/master/etc/examples/notebooks/magic-tutorial.ipynb